container/tabs.go

```diff
@@ -278,14 +293,13 @@ func (r *baseTabsRenderer) moveIndicator(pos fyne.Position, siz fyne.Size, anima
 		r.sizeAnimation = nil
 	}
 
-	r.indicator.Show()
-	if r.indicator.Position().IsZero() {
-		r.indicator.Move(pos)
-		r.indicator.Resize(siz)
-		r.indicator.Refresh()
+	if r.indicator.Position().Subtract(pos).IsZero() && r.indicator.Size().Subtract(siz).IsZero() {
```

Comment:
Is this part of the fix as well, or is it an optimisation?
If I read it correctly this is saying, "if we are starting a new animation, that does not have the same ending state as the current one, but starts at the same place, return" - It seems like we could accidentally not start an animation that should be started in this case..?

Reply:
Oh my mistake, that line is not needed (it was part of my first try to solve the crash). I will delete it.

Reply:
Removed useless condition. Done.